pkg/ephemeral-disk-utils/utils.go

```diff
@@ -66,11 +66,9 @@ func (om *OwnershipManager) SetFileOwnership(file string) error {
 	return os.Chown(file, uid, gid)
 }
 
-func RemoveFile(path string) error {
+func RemoveAllAndLogFailure(path string) error {
 	err := os.RemoveAll(path)
```

Comment:
Now I am confuesed. I don't understand why `os.RemoveAll` was used here in the first place. I would not expect that a non-empty directory gets recursively deleted when I invoke `RemoveFile`.

Reply:
I share your opinion, hence find the name change doubly important.

But frankly, I don't even see much value with logging failures. Logging inside a helper function is a dubious practice. I have a feeling that this function could be replaced by `os.RemoveAll()` in any of its calling places, but stayed conservative in this PR.

Reply:
I modified the function semantics your suggestion.